tests/scripts/thread-cert/Cert_6_6_02_KeyIncrementRollOver.py

```diff
@@ -73,6 +77,54 @@ def test(self):
         for addr in addrs:
             self.assertTrue(self.nodes[LEADER].ping(addr))
 
+    def verify(self, pv):
+        pkts = pv.pkts
+        pv.summary.show()
+
+        LEADER = pv.vars['LEADER']
+        ED = pv.vars['ED']
+        _leader_pkts = pkts.filter_wpan_src64(LEADER)
+        _ed_pkts = pkts.filter_wpan_src64(ED)
+
+        # Step 1: The DUT must start the network using
+        # thrKeySequenceCounter = 127
+        _leader_pkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next().must_verify(
+            lambda p: p.wpan.aux_sec.key_source == 127)
+
+        # Step 2: Verify that the topology described above is created.
+        # MLE Auxiliary security header shall contain Key Source = 127,
+        # KeyIndex = 128, KeyID Mode = 2
+        _ed_pkts.filter_mle_cmd(
+            MLE_CHILD_ID_REQUEST).must_next().must_verify(lambda p: p.wpan.aux_sec.key_index == 128 and p.wpan.aux_sec.
+                                                          key_id_mode == 2 and p.wpan.aux_sec.key_source == 127)
+
+        # Step 3: Leader send an ICMPv6 Echo Request to DUT.
+        # The MAC Auxiliary security header must contain
+        # KeyIndex = 128, KeyID Mode = 1
+        lp = _leader_pkts.filter_ping_request().filter(
+            lambda p: p.wpan.aux_sec.key_index == 128 and p.wpan.aux_sec.key_id_mode == 1).must_next()
+
+        # Step 4: DUT send an ICMPv6 Echo Reply to Leader.
+        # The MAC Auxiliary security header must contain
+        # KeyIndex = 128, KeyID Mode = 1
+        _ed_pkts.filter_ping_reply(identifier=lp.icmpv6.echo.identifier).must_next().must_verify(
+            lambda p: p.wpan.aux_sec.key_index == 128 and p.wpan.aux_sec.key_id_mode == 1)
+
+        # Step 5: Leader increment thrKeySequenceCounter by 1 to force a key switch.
+        _leader_pkts.range(_ed_pkts.index).filter_mle_cmd(MLE_ADVERTISEMENT).must_next()
```

Comment:
need to check `thrKeySequenceCounter`?

Reply:
sometimes the MLE ADVERTISEMENT send in the fist several ping request packets, so the thrKeySequenceCounter will not change. Delete the verification, since the subsequent ping requests are sent
with KeyIndex = 1

Reply:
But we can add `thrKeySequenceCounter` condition to the filters rather than in verify?

Reply:
yes, for ping request, put thrKeySequenceCounter in filter. For the ping reply, put thrKeySequenceCounter in verify